airflow/providers/google/provider.yaml

```diff
@@ -85,6 +85,7 @@ dependencies:
   - google-cloud-build>=3.0.0
   - google-cloud-compute>=0.1.0,<2.0.0
   - google-cloud-container>=2.2.0,<3.0.0
+  - google-cloud-dataflow-client>=0.5.2,<0.5.5
```

Comment:
Any reason this version range is so specific? What happens in 0.5.5?

Reply:
Version 0.5.5 requires higher versions of protobuf and proto-plus libraries which can break other dependencies in google provider package

Reply:
Typical Google. Thanks for the clarification.

Reply:
Can you please add a comment there why ? I know it's not there for the others but we have generally adopted the approach   that whenever we upper-bound a dependency we SHOULD add a comment specifiying why the upper-binding is there and what is the condition for it to be removed. 

https://github.com/apache/airflow#approach-to-dependencies-of-airflow :

> Whenever we upper-bound such a dependency, we should always comment why we are doing it - i.e. we should have a good reason why dependency is upper-bound. And we should also mention what is the condition to remove the binding.

Reply:
@potiuk, is comment in provider.yaml will be enough or project has another place for upper-bound comments?

Reply:
Yep. Comment in provider.yaml is exactly what I meant